src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const smallTerm = 3;
+  const smallTermDiscount = 20;
+  const valueDay = 40;
+
+  if (days >= longTerm) {
+    return days * valueDay - longTermDiscount;
+  }
+
+  if (days >= smallTerm) {
+    return days * valueDay - smallTermDiscount;
+  }
```

Comment:
The current logic applies a discount for rentals longer than 7 days first, and then for rentals longer than 3 days. This is correct, but ensure that the discounts are not cumulative. If the intention is to apply only one discount (the larger one), the current implementation is correct. If the discounts should be cumulative, the logic needs to be adjusted.